modules/cassandra/src/test/java/org/testcontainers/containers/CassandraContainerTest.java

```diff
@@ -0,0 +1,117 @@
+package org.testcontainers.containers;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.CassandraQueryWaitStrategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Eugeny Karpov
+ */
+@Slf4j
+public class CassandraContainerTest {
+
+    private static final String TEST_CLUSTER_NAME_IN_CONF = "Test Cluster Integration Test";
+
+    @Test
+    public void testSimple() throws Exception {
```

Comment:
All `throws Exception` declarations are unecessary in this test class.

Reply:
Yes, you're right. It's just my Intellij Idea template for test methods. 
Need to change it :)